dotenv-sample

```diff
@@ -1,5 +1,6 @@
 # The name of this backend
-BACKEND=expectations
+BACKEND=local
```

Comment:
Should this be `BACKEND=dummy`?

Reply:
It can be anything really; for local envs it doesn't matter, and tests set it explicitly when they need to. In an imminent PR it's going to need to be one of a fixed list, because we'll need to know the backends to iterate over and fetch from job-server, so it may change then anyway